config/defaults.mk

```diff
@@ -486,8 +486,8 @@ ALGOIM_LIB = $(BLITZ_LIB)
 
 # BENCHMARK library configuration
 BENCHMARK_DIR = @MFEM_DIR@/../google-benchmark
-BENCHMARK_OPT = -I$(BENCHMARK_DIR)/include
-BENCHMARK_LIB = -L$(BENCHMARK_DIR)/lib -lbenchmark -lpthread
+BENCHMARK_OPT = -I$(BENCHMARK_DIR)/include -I$(BENCHMARK_DIR)/build/include
+BENCHMARK_LIB = -L$(BENCHMARK_DIR)/build/src -lbenchmark -lbenchmark_main -lpthread
```

Comment:
The `build/*` triggers warnings: `ld: warning: directory not found for option '-L../../../google-benchmark/build/src'`.

Reply:
Is there a need for `-lbenchmark_main` ?

Reply:
I think you're using an old version of `google-benchmark`.

Reply:
Reverted in [aacc37c](https://github.com/mfem/mfem/pull/3105/commits/aacc37c1756641adfc9ddd9bbd51dc0106756cbc).